### PR TITLE
Detect 10, repo updates, and renaming

### PR DESCRIPTION
**Pull request by andrian-sevastyanov**
Preparation for Detect 10:
- repository updates
- name changes

### CODE REVIEW

**Review by niravrsynopsys — APPROVED** [review]

File: build.gradle [hunk 1]
```diff
@@ -35,7 +35,7 @@ apply plugin: 'com.synopsys.integration.simple'
 
 jar {
     manifest {
-        attributes 'Main-Class': 'com.synopsys.integration.synopsys.detect.scripts.Application'
+        attributes 'Main-Class': 'com.blackduck.integration.blackduck.detect.scripts.Application'
```
Comment (gopannabd) [inline build.gradle:38]: Should be 'com.blackduck.integration.detect.scripts.Application'

File: src/main/java/com/blackduck/integration/blackduck/detect/scripts/Application.java [hunk 2]
```diff
@@ -5,18 +5,17 @@
  *
  * Use subject to the terms and conditions of the Synopsys End User Software License and Maintenance Agreement. All rights reserved worldwide.
  */
-package com.synopsys.integration.synopsys.detect.scripts;
+package com.blackduck.integration.blackduck.detect.scripts;
```
Comment (gopannabd) [inline src/main/java/com/blackduck/integration/blackduck/detect/scripts/Application.java:8]: Should be com.blackduck.integration.detect.scripts

File: src/main/resources/earlierversions/detect9.ps1 [hunk 3]
```diff
@@ -246,11 +232,11 @@ function Get-DetectJar ($DetectFolder, $DetectSource, $DetectVersionKey, $Detect
 
     if ($DetectSource -eq "") {
         if ($DetectVersion -eq "") {
-            $DetectVersionUrl = "https://sig-repo.synopsys.com/api/storage/bds-integrations-release/com/synopsys/integration/synopsys-detect?properties=" + $DetectVersionKey
+            $DetectVersionUrl = "https://repo.blackduck.com/api/storage/bds-integrations-release/com/synopsys/integration/synopsys-detect?properties=" + $DetectVersionKey
             $DetectSource = Receive-DetectSource -ProxyInfo $ProxyInfo -DetectVersionUrl $DetectVersionUrl -DetectVersionKey $DetectVersionKey
         }
         else {
-            $DetectSource = "https://sig-repo.synopsys.com/bds-integrations-release/com/synopsys/integration/synopsys-detect/" + $DetectVersion + "/synopsys-detect-" + $DetectVersion + ".jar"
+            $DetectSource = "https://repo.blackduck.com/bds-integrations-release/com/synopsys/integration/synopsys-detect/" + $DetectVersion + "/synopsys-detect-" + $DetectVersion + ".jar"
```
Comment (gopannabd) [inline src/main/resources/earlierversions/detect9.ps1:239]: synopsys-detect -> detect>

File: src/test/java/scripts/ShellScriptTest.java [hunk 4]
```diff
@@ -73,7 +73,7 @@ public File getScriptFile() {
 
     @Test
     void testInnerSpaces() throws IOException, InterruptedException {
-        final boolean success = testEscapingSpaces("--detect.project.name=Synopsys Detect");
+        final boolean success = testEscapingSpaces("--detect.project.name=Blackduck Detect");
```
Comment (gopannabd) [inline src/test/java/scripts/ShellScriptTest.java:76]: Just Detect

Comment (andrian-sevastyanov) [reply src/test/java/scripts/ShellScriptTest.java:76]: I decided to keep it this way as this is a test and it's actually testing specifically that the space between two words doesn't break anything.

File: src/main/resources/templates/detect-sh.sh [hunk 5]
```diff
@@ -116,18 +116,18 @@ run() {
...
+      VERSION_CURL_CMD="curl ${DETECT_CURL_OPTS} --silent --header \"X-Result-Detail: info\" '${DETECT_BINARY_REPO_URL}/api/storage/bds-integrations-release/com/blackduck/integration/detect?properties=${DETECT_VERSION_KEY}'"
       VERSION_EXTRACT_CMD="${VERSION_CURL_CMD} | grep \"${DETECT_VERSION_KEY}\" | sed 's/[^[]*[^\"]*\"\([^\"]*\).*/\1/'"
       DETECT_SOURCE=$(eval ${VERSION_EXTRACT_CMD})
       if [[ -z "${DETECT_SOURCE}" ]]; then
         echo "Unable to derive the location of ${DETECT_VERSION_KEY} from response to: ${VERSION_CURL_CMD}"
         USE_LOCAL=1
       fi
     else
-      DETECT_SOURCE="${DETECT_BINARY_REPO_URL}/bds-integrations-release/com/synopsys/integration/synopsys-detect/${DETECT_RELEASE_VERSION}/synopsys-detect-${DETECT_RELEASE_VERSION}.jar"
+      DETECT_SOURCE="${DETECT_BINARY_REPO_URL}/bds-integrations-release/com/blackduck/integration/detect/${DETECT_RELEASE_VERSION}/detect-${DETECT_RELEASE_VERSION}.jar"
```
Comment (gopannabd) [inline src/main/resources/templates/detect-sh.sh:130]: Thanks for pulling in my MR. ^_^

**Review by gopannabd — APPROVED** [review]

**Pull request merged**